src/app/components/SearchResults/SearchHeader.jsx

```diff
@@ -0,0 +1,163 @@
+    const newPage = Number(pageNumber) - 1;
+    const perPage = searchQuery.per_page;
+    const newQuery = Object.assign({}, searchQuery, { page: newPage, per_page: perPage, total: metadata.total });
+    userQuery(newQuery);
+    submit(newQuery);
+  };
+
+  // click and navigate to any page
+  const navigateToPage = (e, pageNumber) => {
+    e.preventDefault();
+    e.stopPropagation();
+    let page = pageNumber;
+    if (page < 1) {
+      page = 1;
+    }
+    if (page > totalPages) {
+      page = totalPages;
+    }
+    goToPage(page);
+  };
+
+  // update page in store when select of pages change
+  const onChangePage = (e) => {
+    goToPage(Number(e.target.value));
+  };
+
+  // update per_page in store when select of per_ppage changes
+  const onChangePerPage = (e) => {
+    const newPage = 0;
+    const newPerPage = e.target.value;
+    if (newPerPage !== searchQuery.per_page) {
+      const newQuery = Object.assign({}, searchQuery, { page: newPage, per_page: newPerPage, total: metadata.total });
+      userQuery(newQuery);
+      submit(newQuery);
+    }
+  };
+
+  return (
+    <form className="usa-form grid-container padding-0 search-header">
+      <div className="grid-row">
+        <div className="grid-col">Items per page</div>
+        <div className="grid-col">Sort By</div>
+        <div className="grid-col">&nbsp;</div>
+        <div className="grid-col">Page</div>
+        <div className="grid-col">&nbsp;</div>
+      </div>
+      <div className="grid-row">
+        <Select
+          id="items-by-page"
+          selectClass="sfr-select-input usa-select"
+          className="grid-col nypl-search-input"
+          options={[10, 20, 50, 100]}
+          label=""
+          labelClass="usa-label"
+          value={searchQuery.per_page}
+          onChange={onChangePerPage}
+          onBlur={onChangePerPage}
+        />
+        <Select
+          id="sort-by"
+          selectClass="sfr-select-input usa-select"
+          className="grid-col nypl-search-input"
+          options={['Relevance', 'Alphabetically']}
+          label=""
+          labelClass="usa-label"
+        />
+        <div className="grid-col sfr-header-border text-right">
+          <a
+            onClick={e => navigateToPage(e, 1)}
+            onKeyPress={e => navigateToPage(e, 1)}
+            role="link"
+            tabIndex={0}
+            className="margin-x-2"
+          >
+            First
+          </a>
+          <a
+            onClick={e => navigateToPage(e, Number(searchQuery.page))}
+            onKeyPress={e => navigateToPage(e, Number(searchQuery.page))}
+            role="link"
+            tabIndex={0}
+            className="margin-x-2"
+          >
+            Previous
+          </a>
+        </div>
+        <Select
+          id="page-select"
+          selectClass="sfr-select-input usa-select"
+          className="grid-col nypl-search-input"
+          options={pageList}
+          label=""
```

Comment:
There is an empty form label on the page. Is it being generated from this empty prop? The empty label is an a11y error so we need a value here--although what would probably be better is to make the column header ("Page") the label. 

This seems to be true for all the `<select>`s

Reply:
Agree that form elements need to have labels and a corresponding class that hides them visually. I believe there's a default class `usa-sr-only` for hiding elements for screen readers to pick up. We might want to define that class as the default prop for form labels.

Reply:
Stripped down we've got:

    <form>
      <div>
        <div>Items per page</div>
        <div>Sort By</div>
        <div>&nbsp;</div>
        <div>Page</div>
        <div>&nbsp;</div>
      </div>
      <div>
        <div>
          <label for="items-by-page"></label>
          <select id="items-by-page"></select>
        </div>
      </div>
    </form>

An the `<label>` is output as part of the `Select` component? Generating a label and hiding it is kind of a workaround when what I think we really mean is something like:

    <form>
      <div>
        <!-- This is the label -->
        <div><label for="items-by-page">Items per page</label></div>
        <div>Sort By</div>
        <div>&nbsp;</div>
        <div>Page</div>
        <div>&nbsp;</div>
      </div>
      <div>
        <div>
          <!-- So no label needed here -->
          <select id="items-by-page"></select>
        </div>
      </div>
    </form>

If we visually hide the label, then a screenreader will announce it twice: once for what's in the `<div>` and once for what's in the `<label>`

(Is this getting out-of-scope for this particular PR?)

Reply:
Made some changes so to have a label, but hidden. Anyway we must test accesibility with some tool, as with lighthouse this approach doesn't work

Reply:
Am mainly following the guidelines here: https://v2.designsystem.digital.gov/components/form-controls/#dropdown -- We should add some label that makes sense for form inputs. Here we could add something like "Select an index" and visually hide using the Design System's default CSS class `usa-sr-only`

Reply:
Yes that's changed

Reply:
Now it works good for accesibility

Reply:
I got to make everythong 100% accesible (passing in lighthouse), changing some things in Select and TextInput